### ISSUE TITLE
Should suppress complaint about `transition` when `transition-property` is overridden

### ISSUE THREAD

**Issue by andersk** [kristerkari/stylelint-high-performance-animation#216]
Currently on

```css
.widget {
  transition: 0.5s cubic-bezier(0.45, 0.05, 0.55, 0.95);
  transition-property: transform, opacity, visibility;
}
```

this rule complains on the `transition` line:

```
  2:15  ✖  Unexpected use of low performance transition property (all).  plugin/no-low-performance-animation-properties
```

But the shorthand `transition` is immediately overridden with an explicit `transition-property`, so this is a false positive that should be suppressed.

Otherwise we’re forced to write something more verbose, e.g.

```css
.widget {
  transition:
    transform 0.5s cubic-bezier(0.45, 0.05, 0.55, 0.95),
    opacity 0.5s cubic-bezier(0.45, 0.05, 0.55, 0.95),
    visibility 0.5s;
}
```

or

```css
.widget {
  transition-property: transform, opacity, visibility;
  transition-duration: 0.5s;
  transition-timing-function: cubic-bezier(0.45, 0.05, 0.55, 0.95);
}
```
